docs/features/writing-different-telemetry-types.md

```diff
@@ -50,14 +50,14 @@ var request = new HttpRequestMessage(HttpMethod.Post, "http://requestbin.net/r/u
 };
 
 // Start measuring
-var startTime = DateTimeOffset.UtcNow;
-durationMeasurement.Start();
-
-// Send request to dependant service
-var response = await httpClient.SendAsync(request);
-
-_logger.LogHttpDependency(request, response.StatusCode, startTime, durationMeasurement.Elapsed, telemetryContext);
-// Output: "HTTP Dependency requestbin.net for POST /r/ujxglouj completed with 200 in 00:00:00.2521801 at 03/23/2020 09:56:31 +00:00 (Successful: True - Context: [Tenant, Contoso])"
+using (var measurement = DependencyMeasurement.Start())
+{
+    // Send request to dependant service
+    var response = await httpClient.SendAsync(request);
+    
+    _logger.LogHttpDependency(request, response.StatusCode, measurement.StartTime, measurement.Elapsed, telemetryContext);
```

Comment:
Same here, overload

Reply:
☝️ we should show how you can use our overload which just takes in the measurement rather than passing the fields itself

Reply:
☝️ we should show how you can use our overload which just takes in the measurement rather than passing the fields itself